share/python/procedures/proc.py

```diff
@@ -3556,7 +3563,7 @@ def run_detcas(name, **kwargs):
                 raise ValidationError("Second-order MCSCF: PK algorithm only supports C1 symmetry.")
```

Comment:
A nice test case for non-symmetric cases would be to try out second order MCSCF and SCF with the PK object. Keywords are `soscf true` and `so_mcscf true`, respectively.